Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml

```diff
@@ -913,28 +752,111 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: d2b682c2-7df5-4ebd-855a-8949333649fb
+    type: playbook
+    task:
+      id: d2b682c2-7df5-4ebd-855a-8949333649fb
+      version: -1
+      name: Detonate URL - WildFire v2.2
+      description: |-
+        Detonate a webpage or remote file using the WildFire v2 integration. This playbook returns relevant reports to the War Room and file reputations to the context data.
+        The detonation supports the following file types:
+        APK, JAR, DOC, DOCX, RTF, OOXLS, XLSX, PPT, PPTX, XML, PE32, PDF, DMG, PKG, RAR, 7Z, JS.
+      playbookName: Detonate URL - WildFire v2.2
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      Interval:
+        simple: "1"
+      Timeout:
+        simple: "8"
+      URL:
+        complex:
+          root: inputs.URL
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 156ee7d3-6235-4011-85fe-32e8d2154de1
+    type: regular
+    task:
+      id: 156ee7d3-6235-4011-85fe-32e8d2154de1
+      version: -1
+      name: Detonate URL - JoeSecurity
```

Comment:
Don't we use a subplaybook for that?

Reply:
We don't have a sub-playbook for that. It is deprecated and it said to use a command instead. The problem is that in this case we are not checking if the integration is working.